src/model/guild/audit_log.rs

```diff
@@ -467,7 +467,7 @@ impl<'de> Deserialize<'de> for AuditLogs {
             Webhooks,
             #[serde(rename = "users")]
             Users,
-            // TODO(field added by Discord, undocumented) #[serde(rename = "integrations")] Integrations,
+            /* TODO(field added by Discord, undocumented) #[serde(rename = "integrations")] Integrations, */
```

Comment:
This must be reversed as it does not conform to our rustfmt style configuration.
```suggestion
            // TODO(field added by Discord, undocumented) #[serde(rename = "integrations")] Integrations,
```

Reply:
Fixed.